src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -40,6 +41,9 @@
          * to think about number formatting issues.
          */
         Utils.forceJVMLocaleToUSEnglish();
+
+        // Set config factory to know about the configuration options that we have.
```

Comment:
So that means that forcing the JVM locale to use English is not forced in downstream projects. I will open an issue for that one, because downstream projects are unaware of this.